rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -68,12 +144,49 @@ components:
     Accepted:
       description: The transformation request was accepted for processing.
```

Comment:
✏️ Include a note in this description that the status URI in included in the header of the response so that info ends up in the TransformApi class.